draft-js-dnd-plugin/src/Image/index.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import Draggable from '../components/block-draggable-wrapper';
+import Alignment from '../components/block-alignment-wrapper';
+
+class Image extends Component {
+  remove = (event) => {
+    event.preventDefault();
+    event.stopPropagation();
+
+    this.props.blockProps.onRemove(this.props.block.getKey());
+  };
+
+  alignLeft = () => {
+    this.props.align('left');
+  };
+
+  alignCenter = () => {
+    this.props.align('center');
+  };
+
+  alignRight = () => {
+    this.props.align('right');
+  };
+
+  render() {
+    const { blockProps, block, attachButtons, theme, alignment, onDragStart, draggable } = this.props;
+
+    const buttons = [
+      <span className={ theme.get('imageButton') }
+        onClick={this.alignLeft}
+        style={{ marginLeft: '-2.4em' }}
```

Comment:
What about making a theme property: leftImageButton, centerImageButton, rightImageButton. This case it's more flexible. The hard-coded marginLeft -2.4em might not work for other people.